src/rgtk.rs

```diff
@@ -60,8 +60,9 @@ extern crate "cairo-sys" as cairo_ffi;
 
 extern crate glib;
 
+pub use glib::Type;
+pub use glib::{GValue, GValuePublic};
```

Comment:
I see. I assumed that the rgtk crate was supposed to bundle everything, so people didn't have to add multiple Cargo.toml entries. It would be strange, for example, to require a separate dependency just to get the `Type` enum for creating a tree view. I just pushed a possible solution.